doc/releases/release-notes-4.0.rst

```diff
@@ -289,6 +293,9 @@ Drivers and Sensors
 
 * ADC
 
+  * Added proper ADC2 calibration entries in ESP32.
```

Comment:
```suggestion
  * Added proper ADC2 calibration entries in ESP32.
```
 Just caught this one sorry! Good to go otherwise from my point of view, great stuff :)
Don't fix unless there's other more substantial changes requested by others, we can do a global cleanup closer to release date

Reply:
I can't spot the diff in here.. what is that?

Reply:
Oops I had a cached version starting with "Add ..."  on my phone, sorry